eos-tech-support/eos-convert-system-qa

```diff
@@ -93,16 +93,14 @@ if $OSTREE || $APPS; then
 
     # Change the Flatpak runtime and apps server URLs.
     if $APPS; then
-        runtime_url=$(flatpak remote-list -d | awk '/^eos-runtimes/{print $3}')
-        runtime_repo=${runtime_url##*/}
-        new_runtime_url="https://${OSTREE_USER}:${OSTREE_PASSWORD}@${OSTREE_HOST}/${OSTREE_DEV_ROOT}/${runtime_repo}"
+        RUNTIME_REPO=eos-amd64
```

Comment:
I think we should just improve the parsing. We can also use ostree to get it directly: `ostree config get 'remote "eos-runtimes".url'`.